tests/test_models/test_engine/test_db_storage.py

```diff
@@ -78,11 +79,33 @@ def test_all_returns_dict(self):
     @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
     def test_all_no_class(self):
         """Test that all returns all rows when no class is passed"""
+        pass
 
     @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
     def test_new(self):
         """test that new adds an object to the database"""
+        pass
 
     @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
     def test_save(self):
         """Test that save properly saves objects to file.json"""
+        pass
+
+    @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
+    def test_get(self):
+        """Test case for get method at state"""
+        dic_new_state = {'name': 'Valle'}
+        new_obj = State(**dic_new_state)
+        new_obj.save()
+        get_obj = storage.get(State, new_obj.id)
+        self.assertEqual(type(get_obj), State)
+
+    @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
+    def test_count(self):
+        """ Test case for count method at state"""
+        first_count = storage.count(State)
+        dic_new_state = {'name': 'Valle'}
+        new_obj = State(**dic_new_state)
+        new_obj.save()
+        second_count = storage.count(State)
```

Comment:
How is this different from first count?

Reply:
the first count is before create a new state, and the second count is after create it, so we can compare it to verify if the counter increase

Reply:
Ok, its clear now, thx!